packages/commuter-server/src/resources/nteract.metadata.schema.json

```diff
@@ -0,0 +1,18 @@
+{
+    "nteract": {
+        "properties": {
+            "tags": {
+                "type": "string",
+                "description": "Meta data about the notebook"
+            },
+            "description": {
+                "description": "Short description about the notebook",
+                "type": "string"
+            },
+            "image": {
+                "description": "Display image url",
+                "type": "string"
+            }
+        }
+    }
+}
```

Comment:
Cool, we need to move this into nteract/nteract as well. We can probably make a new package there for notebook metadata, cell metadata, and output metadata.